.gitignore

```diff
@@ -59,3 +59,4 @@ typings/
 
 # next.js build output
 .next
+.vscode/settings.json
```

Comment:
nitpick - add a few newlines and a descriptive comment (`# VS Code settings` would suffice)

Reply:
done